examples/deployments/aws-terraform/chroma.tf

```diff
@@ -0,0 +1,168 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+# Define provider
+variable "AWS_ACCESS_KEY" {}
+variable "AWS_SECRET_ACCESS_KEY" {}
+
+provider "aws" {
+  access_key = var.AWS_ACCESS_KEY
+  secret_key = var.AWS_SECRET_ACCESS_KEY
+  region     = var.region
+}
+
+# Create security group
+resource "aws_security_group" "chroma_sg" {
+  name        = "chroma-cluster-sg"
+  description = "Security group for the cluster nodes"
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  dynamic "ingress" {
+    for_each = var.public_access ? [1] : []
+    content {
+      from_port   = 8000
+      to_port     = 8000
+      protocol    = "tcp"
+      cidr_blocks = ["0.0.0.0/0"]
+    }
+  }
+  #
```

Comment:
nit: commented out code?

Reply:
Story of my life ... "commented out code" :D. Removed !